src/common/Backend.ts

```diff
@@ -173,6 +174,10 @@ export class Backend {
     pythonInfo(): Promise<PythonInfo> {
         return this.fetchJson('/python-info', 'GET');
     }
+
+    dependencies(): Promise<{ name: string; dependencies: PyPiPackage[]; description: string }[]> {
```

Comment:
> { name: string; dependencies: PyPiPackage[]; description: string }

Let's add a type for this. Maybe `ChainnerPackage`?

Reply:
Oh, you called it `Package` in dependencies.ts. That works too. Please use the type.